utils/cargo-tfhe-lints-inner/src/main.rs

```diff
@@ -27,6 +27,9 @@ fn main() {
         (tool_args.as_slice(), &[] as &[String])
     };
 
+    std::env::set_var("CARGO_PKG_VERSION_MAJOR", "X");
+    std::env::set_var("CARGO_PKG_VERSION_MINOR", "Y");
```

Comment:
A comment would help understand why this is needed

Reply:
I added a small comment to explain